src/librustc_middle/query/mod.rs

```diff
@@ -932,10 +1020,16 @@ rustc_queries! {
             storage(ArenaCacheSelector<'tcx>)
             desc { "dllimport_foreign_items" }
         }
-        query is_dllimport_foreign_item(_: DefId) -> bool {}
-        query is_statically_included_foreign_item(_: DefId) -> bool {}
-        query native_library_kind(_: DefId)
-            -> Option<NativeLibKind> {}
+        query is_dllimport_foreign_item(def_id: DefId) -> bool {
+            desc { "is_dllimport_foreign_item" }
+        }
+        query is_statically_included_foreign_item(def_id: DefId) -> bool {
+            desc { "is_statically_included_foreign_item" }
+        }
+        query native_library_kind(def_id: DefId)
+            -> Option<NativeLibKind> {
+            desc { "native_library_kind" }
+        }
```

Comment:
I wonder if we should render these like `"foo({})", tcx.def_path_str(def_id)`.

Some of the ones you added could probably also be like this, but it's hard to judge.